<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmGPSData</class>
 <widget class="QMainWindow" name="frmGPSData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>530</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>940</width>
    <height>530</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GPS Data</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resources/FormIcons/resources.qrc">
    <normaloff>:/plugins/ElectricSystems/Resources/FormIcons/formgps.png</normaloff>:/plugins/ElectricSystems/Resources/FormIcons/formgps.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>921</width>
      <height>86</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>87</x>
       <y>21</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Project</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtPro">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>18</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>21</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Database</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>87</x>
       <y>51</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Substation</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>51</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Feeder</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtSub">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>48</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtPBS">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>326</x>
       <y>18</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtFed">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>326</x>
       <y>48</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string/>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>12</y>
       <width>62</width>
       <height>62</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/plugins/ElectricSystems/Resources/FormIcons/basicnreca.png);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="cmdGPSSave">
     <property name="geometry">
      <rect>
       <x>692</x>
       <y>23</y>
       <width>50</width>
       <height>40</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Save</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../Resources/FormIcons/resources.qrc">
       <normaloff>:/plugins/ElectricSystems/Resources/FormIcons/textsave.png</normaloff>:/plugins/ElectricSystems/Resources/FormIcons/textsave.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="cmdCreateFeature">
     <property name="geometry">
      <rect>
       <x>747</x>
       <y>23</y>
       <width>50</width>
       <height>40</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Create Fetaure</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../Resources/FormIcons/resources.qrc">
       <normaloff>:/plugins/ElectricSystems/Resources/FormIcons/featurecreate.png</normaloff>:/plugins/ElectricSystems/Resources/FormIcons/featurecreate.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="cmdHelp">
     <property name="geometry">
      <rect>
       <x>802</x>
       <y>23</y>
       <width>50</width>
       <height>40</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Help</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../Resources/FormIcons/resources.qrc">
       <normaloff>:/plugins/ElectricSystems/Resources/FormIcons/basichelp.png</normaloff>:/plugins/ElectricSystems/Resources/FormIcons/basichelp.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="cmdClose">
     <property name="geometry">
      <rect>
       <x>857</x>
       <y>23</y>
       <width>50</width>
       <height>40</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Close</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../Resources/FormIcons/resources.qrc">
       <normaloff>:/plugins/ElectricSystems/Resources/FormIcons/basicexit.png</normaloff>:/plugins/ElectricSystems/Resources/FormIcons/basicexit.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>447</x>
       <y>51</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Date</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtGroup">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>18</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>447</x>
       <y>21</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Group</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtDate">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>48</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="cmdGPSEdit">
     <property name="geometry">
      <rect>
       <x>637</x>
       <y>23</y>
       <width>50</width>
       <height>40</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Edit</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../Resources/FormIcons/resources.qrc">
       <normaloff>:/plugins/ElectricSystems/Resources/FormIcons/textedit.png</normaloff>:/plugins/ElectricSystems/Resources/FormIcons/textedit.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>921</width>
      <height>411</height>
     </rect>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tabPoleData">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <attribute name="icon">
      <iconset resource="../Resources/FormIcons/resources.qrc">
       <normaloff>:/plugins/ElectricSystems/Resources/FormIcons/protabpole.png</normaloff>
       <normalon>:/plugins/ElectricSystems/Resources/FormIcons/protabpole.png</normalon>
       <disabledoff>:/plugins/ElectricSystems/Resources/FormIcons/protabpole.png</disabledoff>
       <disabledon>:/plugins/ElectricSystems/Resources/FormIcons/protabpole.png</disabledon>
       <activeoff>:/plugins/ElectricSystems/Resources/FormIcons/protabpole.png</activeoff>
       <activeon>:/plugins/ElectricSystems/Resources/FormIcons/protabpole.png</activeon>
       <selectedoff>:/plugins/ElectricSystems/Resources/FormIcons/protabpole.png</selectedoff>
       <selectedon>:/plugins/ElectricSystems/Resources/FormIcons/protabpole.png</selectedon>:/plugins/ElectricSystems/Resources/FormIcons/protabpole.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Pole Survey Data</string>
     </attribute>
     <widget class="QTableView" name="dgvPole">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>5</y>
        <width>907</width>
        <height>366</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabLineData">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <attribute name="icon">
      <iconset resource="../Resources/FormIcons/resources.qrc">
       <normaloff>:/plugins/ElectricSystems/Resources/FormIcons/protabline.png</normaloff>
       <normalon>:/plugins/ElectricSystems/Resources/FormIcons/protabline.png</normalon>
       <disabledoff>:/plugins/ElectricSystems/Resources/FormIcons/protabline.png</disabledoff>
       <disabledon>:/plugins/ElectricSystems/Resources/FormIcons/protabline.png</disabledon>
       <activeoff>:/plugins/ElectricSystems/Resources/FormIcons/protabline.png</activeoff>
       <activeon>:/plugins/ElectricSystems/Resources/FormIcons/protabline.png</activeon>
       <selectedoff>:/plugins/ElectricSystems/Resources/FormIcons/protabline.png</selectedoff>
       <selectedon>:/plugins/ElectricSystems/Resources/FormIcons/protabline.png</selectedon>:/plugins/ElectricSystems/Resources/FormIcons/protabline.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Line Survey Data</string>
     </attribute>
     <widget class="QTableView" name="dgvLine">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>5</y>
        <width>907</width>
        <height>366</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabProRes">
     <attribute name="icon">
      <iconset resource="../Resources/FormIcons/resources.qrc">
       <normaloff>:/plugins/ElectricSystems/Resources/FormIcons/protabres.png</normaloff>
       <normalon>:/plugins/ElectricSystems/Resources/FormIcons/protabres.png</normalon>
       <disabledoff>:/plugins/ElectricSystems/Resources/FormIcons/protabres.png</disabledoff>
       <disabledon>:/plugins/ElectricSystems/Resources/FormIcons/protabres.png</disabledon>
       <activeoff>:/plugins/ElectricSystems/Resources/FormIcons/protabres.png</activeoff>
       <activeon>:/plugins/ElectricSystems/Resources/FormIcons/protabres.png</activeon>
       <selectedoff>:/plugins/ElectricSystems/Resources/FormIcons/protabres.png</selectedoff>
       <selectedon>:/plugins/ElectricSystems/Resources/FormIcons/protabres.png</selectedon>:/plugins/ElectricSystems/Resources/FormIcons/protabres.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Processing Result</string>
     </attribute>
     <widget class="QWebView" name="webView">
      <property name="geometry">
       <rect>
        <x>4</x>
        <y>4</y>
        <width>908</width>
        <height>367</height>
       </rect>
      </property>
      <property name="url">
       <url>
        <string>about:blank</string>
       </url>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Resources/FormIcons/resources.qrc"/>
 </resources>
 <connections/>
</ui>
